src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultClaimUidExtractor.cs

```diff
@@ -38,6 +38,17 @@ public string ExtractClaimUid(ClaimsIdentity claimsIdentity)
         // Internal for testing
         internal static IEnumerable<string> GetUniqueIdentifierParameters(ClaimsIdentity claimsIdentity)
         {
+            var subClaim = claimsIdentity.FindFirst(
+                claim => string.Equals("sub", claim.Type, StringComparison.Ordinal));
+            if (subClaim != null && !string.IsNullOrEmpty(subClaim.Value))
+            {
+                return new string[]
+                {
+                    "sub",
+                    subClaim.Value
```

Comment:
This comment https://github.com/aspnet/Antiforgery/issues/30#issuecomment-180546654 mentions the claim and issuer... @blowdart want to explain exactly what to do?